src/features/unified-share-modal/__tests__/SharedLinkSection.test.js

```diff
@@ -185,4 +185,17 @@ describe('features/unified-share-modal/SharedLinkSection', () => {
             expect(wrapper).toMatchSnapshot();
         });
     });
+
+    test('should render disabled create share link message', () => {
+        const sharedLink = { url: '', canChangeAccessLevel: true };
+        const item = { grantedPermissions: { itemShare: false } };
+
+        expect(getWrapper({ sharedLink, item })).toMatchSnapshot();
+    });
+
+    test('should render disabled remove share link message', () => {
```

Comment:
Can we add a description of when this should happen?

Reply:
👍